sysbench/lua-tests/db/common.lua

```diff
@@ -38,7 +38,7 @@ pad CHAR(60) DEFAULT '' NOT NULL,
       query = [[
 CREATE TABLE IF NOT EXISTS sbtest]] .. i .. [[ (
 id SERIAL NOT NULL,
-k INTEGER DEFAULT '0' NOT NULL,
+k bigint DEFAULT '0' NOT NULL,
```

Comment:
```suggestion
k BIGINT DEFAULT '0' NOT NULL,
```
It's Ok, but I think it's better to keep consistent with others.